composer.lock

```diff
@@ -543,25 +543,25 @@
         },
         {
             "name": "symfony/deprecation-contracts",
-            "version": "v3.1.1",
+            "version": "v2.5.2",
```

Comment:
This was probably downgraded, because v3 only supports PHP >=8.0 and your library still supports PHP 7.4 (As Shopware still does that as well).
I ran `composer update` with PHP7.4.